test/index.test.ts

```diff
@@ -346,4 +346,46 @@ describe('transform', () => {
       "
     `)
   })
+
+  it('suupports play functions', async () => {
+    const code = `
+      <template>
+        <Stories>
+          <Story title="Primary", :play="playFunction">
```

Comment:
```suggestion
          <Story title="Primary" :play="playFunction">
```
This is a typo right?